src/api/app/views/webui/request/beta_show_tabs/_overview.html.haml

```diff
@@ -1,6 +1,6 @@
 %h4.list-group.mb-4 Comments & Request History
-.comments-list{ data: { comment_counter: local_assigns[:comment_counter_id] } }
+.timeline{ data: { comment_counter: local_assigns[:comment_counter_id] } }
   = render BsRequestActivityTimelineComponent.new(bs_request: bs_request)
 
-  .comment_new.mt-3
-    = render partial: 'webui/comment/new', locals: { commentable: bs_request }
+.comment_new.mt-3
```

Comment:
This change is really needed because we need to attach different event hooks for the `.timeline` and `.comment_new`